src/storage/txn/scheduler.rs

```diff
@@ -484,6 +487,8 @@ impl<E: Engine, L: LockManager> TxnScheduler<E, L> {
             quota_limiter,
             resource_manager,
             feature_gate,
+            // TODO: Make it configurable
+            txn_status_cache: TxnStatusCache::new(40 * 1000 * 128),
```

Comment:
Can we explain how this is decided?

Reply:
Sorry, I just noticed that some changes seem to be lost from this PR.. I'll update it...